pandas/core/groupby/generic.py

```diff
@@ -1296,6 +1314,23 @@ class DataFrameGroupBy(NDFrameGroupBy):
     A
     1   1   2  0.590716
     2   3   4  0.704907
+
+    To control the output names with different aggregations
+    per column, pass supports "keyword aggregation"
+
+    >>> df.groupby("A").agg(
+    ...     b_min=pd.KeywordAgg(column="B", aggfunc="min"),
+    ...     c_sum=pd.KeywordAgg(column="C", aggfunc="sum"))
+       b_min     c_sum
+    A
+    1      1 -1.956929
+    2      3 -0.322183
+
+    The keywords are the column names, and the values should be
+    2-tuples where the first element is the column selection and
+    the second element is the aggfunc. Pandas provides the
+    ``pandas.KeywordAgg`` namedtuple to clarify the meaning of the values.
+    See :ref:`groupby.aggregate.keyword` for more.
```

Comment:
Suggestions:

`The keywords are the column names` --> `The keywords are the output column names`

and, optionally, to mirror the main documentation which is clearer IMHO:

`the values should be 2-tuples where the first element is the column selection and the second element is the aggfunc`
-->
`the values are 2-tuples whose first element is the column to select and the second element is the aggregation function to apply to that column`